src/graphql/getter.ts

```diff
@@ -68,15 +69,13 @@ export default class Getter extends CommandBase {
     return this;
   };
 
-  withNearText = (nearTextObj: any) => {
+  withNearText = (args: object) => {
```

Comment:
here also, shouldn't be here `NearTexr` instead of `object`?

Reply:
Updated with interface for NearTextArgs